bin/boot_docker

```diff
@@ -39,13 +39,12 @@ $0 [OPTION]
 EOF
 )
 
-# defaults, note there is no default for PROD_DB_PASSWORD, SENDGRID_USERNAME or SENDGRID_PASSWORD
+# defaults, note there is no default for PROD_DB_PASSWORD, SECRET_KEY_BASE, SENDGRID_USERNAME or SENDGRID_PASSWORD
```

Comment:
Why do we not want to have a default here?  the prod password and sendgrids make sense, but it seems like having a default secret key base if one can't be found in the environment is a reasonable thing

Reply:
We can't a default value because it would have to be represented in code as an explicit string - we can't randomly generate one because the whole point is that we're reusing the same value now, and randomly generating one would cause errors up and down the stack.  Even a default value would do the same thing - if you didn't have it set, and then the default got used, any user auth tokens stored in the DB would be encrypted using that value.  Then if you _did_ set the value correctly from vault, it would cause the aforementioned decryption errors.  Storing a default a value as a string in source would also violate several security practices (and `git-secrets` would disallow the commit as well).